tests/integration/scenarios/metaplaylist_dash.js

```diff
@@ -0,0 +1,115 @@
+import { expect } from "chai";
+import sinon from "sinon";
+import RxPlayer from "../../../src";
+import {
+  mockAllRequests,
+  mockManifestRequest,
+} from "../utils/mock_requests.js";
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import of 'mockMetaPlaylistRequest'. ([javascript:S1128](https://sonarcloud.io/organizations/rx-player/rules#rule_key=javascript%3AS1128))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=178&issues=AWLjl606-pl6AHs2a-uU&open=AWLjl606-pl6AHs2a-uU)

[ISSUE_KEY]: AWLjl606-pl6AHs2a-uU